### ISSUE TITLE
Creating more than just 4koma

### ISSUE THREAD

**Issue by TomLucidor** [jbilcke-hf/ai-comic-factory#12]
Nice to see the problem of guided comic generation being tackled. There is a bit of a nit-pick: there are only 1 page and 4 panels, instead of a streamline from prompt to full comic chapter. StoryBird can generate whole stories, allow for narrative tuning and editing, and then generate images accordingly. what would be the major issue preventing having this type of feature? https://storybird.ai/

Side note: for comic generation there are already tools for dynamic panels, which is worth checking, since going from storyline to deciding on aspect ratio of panel outputs look strong https://shortbread.ai/

**Comment by jbilcke-hf** [comment]
hey! sorry for the late reply!

> StoryBird can generate whole stories, allow for narrative tuning and editing, and then generate images accordingly.
> what would be the major issue preventing having this type of feature?

The AI Comic Factory can generate more than 1 page, but this is currently disabled due to a peak in traffic and it took up too much ressources. I'm working on a feature to enable longer stories and panels.

Regarding tuning and tweaking, today we can edit the prompt on a per-panel basis, but I agree more could be done.

I know the AI Comic Factory is often criticized and [compared to tools like Scenario.com](https://twitter.com/rodrigon/status/1744923377276502322) and the likes ([story.com](https://story.com), [kartoon.ai](https://kartoon.ai), even things remotely similar like [glif](https://glif.app/spheres/comics) or [plugger.ai comics](https://www.plugger.ai/tasks/ai-comic-generator) etc) and maybe it is fair, but those are commercial companies with the money to have an actual development team, while the AI Comic Factory is a tiny open-source project created and maintained by a single person (me), so I generally I don't have the time to copy other people's ideas and features (even if in some case I would like to, as I believe the coolest tools should be made open-source and available to anyone and not paywalled behind a subscription)

**Comment by atulkashyap404** [comment]
yes, but we can make it a great AI comic factory. I love this web app.

**Comment by TomLucidor** [comment]
@jbilcke-hf it is a FOSS project with a lot of potential ngl, it's a daydream that InvokeAI (or some other tool in StabilityMatrix) could have a comic feature
